spec/propertybase_id_spec.rb

```diff
@@ -107,28 +101,24 @@
       end
 
       context "non existing object" do
-        let(:input_id) { "zzz456789012345678" }
+        let(:input_id) { "zzz4567890123456" }
```

Comment:
It doesn't make any difference for the result, but for the tests it's easier to see, that the object is the only field affected